glide.lock

```diff
@@ -0,0 +1,140 @@
+hash: 3f1bc464b9b68652d4ba23049e3e876c4fbbe3930798c1deada6f01d54f30c89
+updated: 2016-09-16T10:40:46.133629323-07:00
+imports:
+- name: github.com/appc/spec
+  version: db96f94ae6b227fe4d8288527ead8927181620f6
+  subpackages:
+  - aci
+  - schema
+  - schema/types
+  - pkg/tarheader
+  - schema/common
+  - pkg/device
+- name: github.com/armon/go-metrics
+  version: 3df31a1ada83e310c2e24b267c8e8b68836547b4
+- name: github.com/asaskevich/govalidator
+  version: 9699ab6b38bee2e02cd3fe8b99ecf67665395c96
+- name: github.com/codegangsta/cli
+  version: 01857ac33766ce0c93856370626f9799281c14f4
+- name: github.com/codegangsta/negroni
+  version: c7477ad8e330bef55bf1ebe300cf8aa67c492d1b
+- name: github.com/coreos/go-semver
+  version: 6fe83ccda8fb9b7549c9ab4ba47f47858bc950aa
+  subpackages:
+  - semver
+- name: github.com/dnsimple/dnsimple-go
+  version: c748e13ef2d1c80e1bdfbb26e4fe127d59afa292
+  subpackages:
+  - dnsimple
+- name: github.com/ghodss/yaml
+  version: c3eb24aeea63668ebdac08d2e252f20df8b6b1ae
+- name: github.com/gogo/protobuf
+  version: 82d16f734d6d871204a3feb1a73cb220cc92574c
+  subpackages:
+  - proto
+- name: github.com/golang/protobuf
+  version: 888eb0692c857ec880338addf316bd662d5e630e
+  subpackages:
+  - proto
+- name: github.com/google/go-querystring
+  version: 9235644dd9e52eeae6fa48efd539fdc351a0af53
+  subpackages:
+  - query
+- name: github.com/hashicorp/go-msgpack
+  version: fa3f63826f7c23912c15263591e65d54d080b458
+  subpackages:
+  - codec
+- name: github.com/hashicorp/memberlist
+  version: a93fbd426dd831f5a66db3adc6a5ffa6f44cc60a
+- name: github.com/intelsdi-x/gomit
+  version: db68f6fda248706a71980abc58e969fcd63f5ea6
+- name: github.com/intelsdi-x/snap-plugin-lib-go
+  version: 49a6947457c6267e97e9516df2907d5abf7a7d79
```

Comment:
I think if this is updated to be master(55956e53732655f4875708b15c25904090af41ec) the build will succeed. I think something was inadvertently added to one of the mock plugins that requires the latest master from the plugin-lib.